spec/index.html

```diff
@@ -243,7 +243,7 @@ <h2>Introduction</h2>
       <p>
         Since YAML is more expressive than JSON,
         both in the available data types and in the document structure
-        (see I-D.ietf-yaml-mediatypes),
+        (see [[I-D.ietf-yaml-mediatypes]]),
```

Comment:
```suggestion
        (see [[I-D.ietf-httpapi-yaml-mediatypes]]),
```

Reply:
What is defined in `localBiblio` is "I-D.ietf-httpapi-yaml-mediatypes". Of course, that could be changed, but we need to be sure it's done consistently throughout the document.

Reply:
correct!